cmd/nvidia-ctk/hook/chmod/chmod.go

```diff
@@ -112,34 +113,44 @@ func (m command) run(c *cli.Context, cfg *config) error {
 		return fmt.Errorf("empty container root detected")
 	}
 
-	paths := m.getPaths(containerRoot, cfg.paths.Value())
+	desiredModeInt, err := strconv.ParseUint(cfg.mode, 8, 32)
```

Comment:
Could we check this in `valdidateFlags` instead?

Reply:
I was thinking about this, but we also need the parsed value here - so we either need to save it after parsing or parse twice.

Reply:
I've pushed a commit where I've moved it into `valdidateFlags` - saving the result in additional cfg field. We can leave it or drop it - up to you :)